src/services/lasnService.js

```diff
@@ -0,0 +1,35 @@
+const {
+  SubscribeRequest,
+  LinkRequest,
+  DisconnectRequest,
+} = require('../../protos/lasn/v1/lasn_pb');
+
+export const getLasnStream = (lasnService, userId) => {
```

Comment:
Wondering if we could call it subscription rather than stream.
```suggestion
export const getLasnSubscription = (lasnService, userId) => {
```

Reply:
tbh, I think is stream better bc we are listening for events(stream of events type of thing). However im down to change it to subscription, lmk.